mappings/net/minecraft/client/realms/gui/screen/RealmsUploadScreen.mapping

```diff
@@ -20,6 +20,8 @@ CLASS net/minecraft/class_4427 net/minecraft/client/realms/gui/screen/RealmsUplo
 	FIELD field_20194 previousTimeSnapshot Ljava/lang/Long;
 	FIELD field_20195 bytesPersSecond J
 	FIELD field_20196 UPLOAD_LOCK Ljava/util/concurrent/locks/ReentrantLock;
+	FIELD field_20503 DOT_TEXTS [Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	FIELD field_20503 dotTexts [Lnet/minecraft/class_2561;
```

Reply:
Fixed in ca637d2ab4e446874f1066f968562d0e9185f398